dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProjectMapper.xml

```diff
@@ -29,7 +29,7 @@
         select
         <include refid="baseSql"/>
         from t_ds_project
-        where code = #{code}
+        where code = #{projectCode}
```

Comment:
The change of variable name is not seen in mapper class, 
Are you sure it doesn't need to be modified.

Reply:
Project queryByCode(@Param("projectCode") Long projectCode);